app/views/user_mailer/already_confirmed.html.slim

```diff
@@ -3,12 +3,15 @@ html
     meta content='text/html; charset=UTF-8' http-equiv='Content-Type'
   body
 
-    p= 'A request was made to resend confirmation instructions to this email.' \
-       ' This email address has already been confirmed. You can sign in with ' \
-       'your login.gov account by clicking the link below:'
+    p
+      | A request was made to resend confirmation instructions to this email.
+        This email address has already been confirmed. You can sign in with
+        your #{APP_NAME} account by clicking the link below:
```

Comment:
We should be consistent in the style we use.   At least some of the other slim files use:

```
p
  'Line 1
  'Line 2
```

I don't mind wihch one we choose if they are the equivalent.  Perhaps a followup change to make all the slim files consistent?  I guess in the long run everything will be localised anyway we shouldn't have text in the slim files at all.